src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -523,7 +545,7 @@ else if (participation.getExercise() instanceof ModelingExercise || participatio
      * Get the latest relevant result from the given participation (rated == true or rated == null) (relevancy depends on Exercise type => this should be overridden by subclasses
      * if necessary)
      *
-     * @param participation the participation whose results we are considering
+     * @param participation           the participation whose results we are considering
```

Comment:
This change doesn't look correct.

Reply:
Thats how intellj formats the file automatically.